recipes/enable_chef_error_handler.rb

```diff
@@ -0,0 +1,13 @@
+
+cookbook_file "#{node['cluster']['scripts_dir']}/write_chef_error_handler.rb" do
+  source 'event_handler/write_chef_error_handler.rb'
```

Comment:
why do you need to create the `write_chef_error_handler.rb` file in the node? 
I mean this file is already part of the cookbook recipes installed with ParallelCluster cookbook. 

Can you just enable the existing file? What am I missing? If I'm missing something let's add a comment to explain why do we need to copy this file.